.yamllint

```diff
@@ -10,4 +10,7 @@ rules:
   brackets:
     max-spaces-inside: 1
     level: error
+  document-start: disable
```

Comment:
almost nobody is using the `---` prefix and adding it would touch a huge amount of files. we can do it, but in another change.